chartmuseum2oci/main.go

```diff
@@ -112,7 +135,69 @@ func initHarborHost() {
 	harborHost = u.Host
 }
 
+// checkHelmVersion checks if Helm version meets the minimum requirement (>= 3.19.0)
```

Comment:
```suggestion
// checkHelmVersion checks if Helm version meets the minimum requirement
```
We may forgot to update the comment when modifying the const value